src/lib/web-component/filter-timestamp/filter-timestamp.css

```diff
@@ -37,4 +49,9 @@
 .start-time-container,
 .stop-time-container {
   display: flex;
+  align-items: center;
+}
+
+[hidden] {
+  display: none;
 }
```

Comment:
@atiqueahmedziad this shouldn't be actually needed, did you had any actual issue that you are workarounding with this?

Reply:
The close icon along with start and stop timestamp does not disappear when until I set this. Below, I shared a screenshot, when I didn't set this CSS.
### screenshot 
<img width="975" alt="Screenshot 2020-08-13 at 12 28 54 AM" src="https://user-images.githubusercontent.com/8364578/90053340-25aa2b80-dcfc-11ea-800d-9099e56eba29.png">

Reply:
@atiqueahmedziad I see, yeah that seems to be because the "hidden attribute is not being applied to the elements inside the shadow DOM" (/me sigh for a couple of seconds :-)), would you mind to add an inline comment right above it to explicitly mention that reason?

Reply:
Yes, I have added :-)